src/main/java/frc/robot/commands/ClimbCommand.java

```diff
@@ -24,7 +25,9 @@ public void initialize() {}
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
+
     if (m_subsystem.getclimbingmode()) {
+      m_subsystem.resetTargetedHeight();
```

Comment:
This seem like it should go in ClimbEnable instead.

Reply:
I agree with Adam, makes more sense to reset when climb is enabled instead of conditionally here. What's the thought for keeping it here?

Reply:
fixed (forgot to git rid of the old one)